oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+      * **Packaged Learning Content** (an "instantiation" of a "Reusable Learning Content" with structure, dates, and 
+        configuration).
+
+  * **Discoverable Course**, a learner-facing representation of a course that entices enrollment with marketing information
+    such as the course's price and credit/degree outcomes.
+
+    * This is what we call **Catalog Course**, for the time being.
+    * It is bound to a specific marketing site.
+
+  * **Learner Consumable Course**, a learner-engaging course, with read-optimized representation of the learning content. 
+
+    * This is what we call **(LMS) Course Run**.
+    * It is bound to a specific learner.
+
+* Given the above distinctions, each `edX subdomain`_ will manage its own concept. Hence, the authoring logic
+  implementation for each concept will also remain separate. This prevents us from needing to maintain these concepts across
+  multiple domains: keep the authoring of Discoverable-Course (**Catalog Course**) within Discovery; keep the authoring of 
+  Learning-Content-Course (**Course Run**) within Content Authoring. For example, concepts of price, marketing-publish-date,
+  marketing sales, discounts, etc., do not need to bleed into the Content Authoring service.
+
+* The following is Course Run metadata and its editing should be owned by the `Content Authoring subdomain`_:
+
+  * course run dates
+
+    * start
+    * end
+    * enrollment start
+    * enrollment end
+    * certificate availability
+
+  * pacing type
+
+* The following is Catalog Course metadata and its editing should be owned by the `Discovery subdomain`_:
+
+  * course about image
+  * instructors
+  * seat price
+  * seat type
+  * other catalog course marketing data
+
+  **Note:** course about image and instructors are currently managed as Course Run metadata but will transition to be Catalog Course metadata. 
+
+* The Discovery service will be the single source of truth for the published version of the above data. Though to address the data
+  synchronization issues, we will limit the editing of a piece of data to its single owning service. Anywhere else that a piece of data is editable
+  will be made read-only.
+
+* We will introduce a new front end for editing Catalog Course metadata, written using the latest recommended front end technologies and leveraging phase 3
+of the `decoupled frontend architecture`_ plan.
+
+* From the course author's perspective, we will provide a unified user experience by having the current Studio tool
+  be the portal to authoring both Course Run metadata and Catalog Course metadata. So even though authoring of Catalog Course
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
+  Authoring and Catalog Course Authoring.
+
+  * This will depend on upcoming architectural infrastructure work on single sign on across the edX microservices and a universal
+    front end header and footer.
```

Comment:
I'd like to see us go with a Phase 4 (independent frontend application) implementation of the authoring UI. We are making good progress on single sign on and should be able to work on the universal header/footer in parallel with this project.